triage_bot/lib/engine.dart

```diff
@@ -0,0 +1,1499 @@
+// Copyright 2019 The Flutter Authors. All rights reserved.
```

Comment:
We are missing docs for most classes and functions.

Reply:
It's all internal API, so I'm not sure what API docs would really make sense. I'm happy to document anything that seems unclear. Can you elaborate on what questions you have about the code?